packages/typo3-docs-theme/assets/js/versions.js

```diff
@@ -137,7 +137,10 @@
         let versionTrimmed = version.trim();
         let versionAsFloat = parseFloat(versionTrimmed);
         if (!isNaN(versionAsFloat) && Number(versionTrimmed) === versionAsFloat) {
-          version = versionAsFloat;
+          version = versionAsFloat.toString();
```

Comment:
What would happen to version 4.10 here? I think it would be reduced to 4.1

Reply:
Yes, that happens.

I yield for now.

Reply:
I found this in a StackOverflow, need to verify:

```
function parseAndPreserveFloat(str) {
    // Parse the float value from the string
    let num = parseFloat(str);
    
    // Check if the original string contains ".0"
    if (str.includes('.') && str.endsWith('.0')) {
        return num.toFixed(1); // Ensures one decimal place
    }
    
    return num.toString(); // Otherwise, return the float as a string
}
```